File: 11_amend_The_Sentence/amendSentence.py
# Amend The Sentence - Adobe - Interview Question -
# in Python

def amendTheSentence(text):
    new_str = ''

    for char in text:
        if (char.isupper()):
            new_str += ' '+ char
        else: new_str += char

    return new_str.strip().lower()

input = "YouAreInTheRightPlace"
print("input: ", input)
print("output: ",amendTheSentence(input))

File: 11_amend_The_Sentence/amendSentence.js
// Amend The Sentence - Adobe - Interview Question -
// in JavaScript

function amendTheSentence(text) {
    var new_str = '';
    
    for (char of text) {
        if (char>="A" && char <= "Z") new_str += ' '+ char;
        else new_str += char;
    }
    return new_str.trim().toLowerCase();
}

let input = "YouAreInTheRightPlace";
console.log("input: ", input);
console.log("output: ", amendTheSentence(input));